java/src/jmri/jmrit/logixng/tools/swing/WhereUsedAction.java

```diff
@@ -0,0 +1,33 @@
+package jmri.jmrit.logixng.tools.swing;
+
+import java.awt.event.ActionEvent;
+
+import jmri.util.swing.JmriAbstractAction;
+
+/**
+ * Swing action to create and register a WhereUsedFrame
+ *
+ * @author Dave Sand Copyright (C) 2020
```

Comment:
Is this the right copyright date?

Reply:
Yes. I just copied the file without doing any changes to it. It's a copy of `jmri.jmrit.whereused.WhereUsedAction`.